docs/source/extensions.rst

```diff
@@ -24,6 +24,14 @@ This allows to manage DNS information for your hosts from the Serveradmin
 Servershell and Remote API (and it's benefits such as authentication and
 authorisation).
 
+This implementation hooks into blocking signals
+(`Django Signals <https://docs.djangoproject.com/en/3.2/topics/signals/>`_)
```

Comment:
In the PR I didn't see any signal firing/receiving. Can you point me to, where the signals are getting used/sent?

Reply:
We will hook into the [post_commit](https://github.com/innogames/serveradmin/blob/master/serveradmin/serverdb/query_committer.py#L121) signal. The receiving is not yet implemented.

Reply:
I see .. then I would also not expect this information here. But it's fair enough.